main.tf

```diff
@@ -63,6 +63,13 @@ resource "aws_db_subnet_group" "default" {
   tags        = "${module.label.tags}"
 }
 
+resource "aws_rds_cluster_parameter_group" "default" {
+  name = "${module.label.id}"
+  family = "aurora5.6"
```

Comment:
Thank you for submitting the changes.

`family` should not probably be hardcoded as it could change anytime.
It's better to use a var `cluster_family` for example with the default value `aurora5.6`.